android/src/main/java/com/reactnativestripesdk/pushprovisioning/TapAndPayProxy.kt

```diff
@@ -0,0 +1,133 @@
+package com.reactnativestripesdk.pushprovisioning
+
+import android.app.Activity
+import android.util.Log
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.bridge.WritableNativeMap
+import com.reactnativestripesdk.createError
+import com.google.android.gms.tasks.Task
+
+object TapAndPayProxy {
```

Comment:
Since google's tap and pay library is distributed as an `.aar`, we cannot use `compileOnly` 😢  so this object uses a a decent amount of reflection to get the job done